lib/bundlex/doxygen/generator.ex

```diff
@@ -0,0 +1,130 @@
+defmodule Bundlex.Doxygen.Generator do
```

Comment:
I would move this file level up, to be consistent with convention, that module `MyModule` defined in `dir/my_module.exs` uses modules defined in `dir/my_module/` directory

Reply:
I do not understand. Do you want `Mix.Tasks.Bundlex.Docs` to use `Bundlex.DoxygenGenerator` instead of `Bundlex.Doxygen.Generator`?

Reply:
It might be, or `Bundlex.Doxygen`